modules/openapi-generator/src/main/resources/JavaJaxRS/resteasy/pojo.mustache

```diff
@@ -1,5 +1,4 @@
 import io.swagger.annotations.*;
-{{#useBeanValidation}}import javax.validation.Valid;{{/useBeanValidation}}
```

Comment:
what if  only `useBeanValidation` is set to true but `useJakartaEe` is set to false (default)? 

what about using the following instead?

```
{{^useJakartaEe}}
{{#useBeanValidation}}
import javax.validation.Valid;
{{/useBeanValidation}}
{{/useJakartaEe}}
```

Reply:
please ignore. I think I found the issue